Xamarin.Forms.Core/Shell/Shell.cs

```diff
@@ -431,7 +431,7 @@ public Task GoToAsync(ShellNavigationState state, bool animate)
 		internal async Task GoToAsync(ShellNavigationState state, bool? animate, bool enableRelativeShellRoutes)
 		{
 			// FIXME: This should not be none, we need to compute the delta and set flags correctly
```

Comment:
Is this comment still accurate?

Reply:
yes

When user requests a new url destination there should be code here to figure out what type of `ShellNavigationSource` that should be

If it's popping pushing etc...